### ISSUE TITLE
Using deprecated classesdir instead of classesdirs

### ISSUE THREAD

**Issue by scphantm** [nebula-plugins/gradle-lint-plugin#148]
Gradle is giving me this warning when i enable stacktrace

```
Gradle now uses separate output directories for each JVM language, but this build assumes a single directory for all classes from a source set. This behaviour has been deprecated and is scheduled t
o be removed in Gradle 5.0
        at org.gradle.api.internal.tasks.DefaultSourceSetOutput.getClassesDir(DefaultSourceSetOutput.java:80)
        at org.gradle.api.internal.tasks.DefaultSourceSetOutput_Decorated.getClassesDir(Unknown Source)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
        at org.codehaus.groovy.runtime.metaclass.MultipleSetterProperty.getProperty(MultipleSetterProperty.java:49)
        at org.gradle.internal.metaobject.BeanDynamicObject$MetaClassAdapter.getProperty(BeanDynamicObject.java:228)
        at org.gradle.internal.metaobject.BeanDynamicObject.tryGetProperty(BeanDynamicObject.java:171)
        at org.gradle.internal.metaobject.CompositeDynamicObject.tryGetProperty(CompositeDynamicObject.java:55)
        at org.gradle.internal.metaobject.AbstractDynamicObject.getProperty(AbstractDynamicObject.java:59)
        at org.gradle.api.internal.tasks.DefaultSourceSetOutput_Decorated.getProperty(Unknown Source)
        at org.codehaus.groovy.runtime.callsite.PogoGetPropertySite.getProperty(PogoGetPropertySite.java:50)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:296)
        at com.netflix.nebula.lint.rule.dependency.DependencyService.sourceSetOutput(DependencyService.groovy:422)
        at com.netflix.nebula.lint.rule.dependency.DependencyService.memoizedMethodPriv$classReferencesString(DependencyService.groovy:163)
```

im using gradle 4.1

**Comment by scphantm** [comment]
Is this project no longer maintained?  I was forced to pull this from my stack because of this issue.  Our system is very strict and we treat warnings as build fails.

**Comment by DanielThomas** [comment]
It's maintained, but maybe we're not using the rule that triggers this (all of our nebula tests fail if they have deprecation warnings). I'll look next week.

**Comment by DanielThomas** [comment]
Fixed by https://github.com/nebula-plugins/gradle-lint-plugin/releases/tag/v8.3.2
